Examples.fshx

```diff
@@ -53,32 +133,154 @@ Instance: mCODEKarnofskyPerformanceStatusExample01
+
+
+Instance: mCODETNMClinicalStageGroupExample01
+InstanceOf: TNMClinicalStageGroup
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMClinicalStageGroup"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21908-9 "Stage group.clinical Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#3C "IIIC"
+* hasMember = Reference(mCODETNMClinicalPrimaryTumorCategoryExample01) and Reference(mCODETNMClinicalRegionalNodesCategoryExample01) and Reference(mCODETNMClinicalDistantMetastasesCategoryExample01)
+
+
+Instance: mCODETNMPathologicDistantMetastasesCategoryExample01
+InstanceOf: TNMPathologicDistantMetastasesCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMPathologicDistantMetastasesCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21901-4 "Distant metastases.pathology [Class] Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#pM0 "M0"
+
+
+Instance: mCODETNMPathologicPrimaryTumorCategoryExample01
+InstanceOf: TNMPathologicPrimaryTumorCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMPathologicPrimaryTumorCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21899-0 "Primary tumor.pathology Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#pT3 "T3"
+
+
+Instance: mCODETNMPathologicRegionalNodesCategoryExample01
+InstanceOf: TNMPathologicRegionalNodesCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMPathologicNodesCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21900-6 "Regional lymph nodes.pathology [Class] Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#pN3 "N3"
+
+
+Instance: mCODETNMPathologicStageGroupExample01
+InstanceOf: TNMPathologicStageGroup
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMPathologicStageGroup"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21902-2 "Stage group.pathology Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#3C "IIIC"
+* hasMember = Reference(mCODETNMClinicalPrimaryTumorCategoryExample01) and Reference(mCODETNMClinicalRegionalNodesCategoryExample01) and Reference(mCODETNMClinicalDistantMetastasesCategoryExample01)
```

Comment:
See previous comment about setting the slices individually using the slice names. Also see previous comment about not mixing Clinical and Pathological.